### DISCUSSION TITLE
Show exact bank balance on bank screen

### DISCUSSION THREAD

**Discussion by Fasidious** [endless-sky/endless-sky#9673] [Q&A] [answered]
Hello

Since there is no way in-game to see exactly how many credits you have once you’re into the millions, I suggest that your exact bank balance be shown on the bank screen.

While there’s little need to know your exact bank balance, it would contribute to immersion and would make data control freaks like me happy!

**Comment by xobes** [comment, marked as answer]
Hover over the credits line in the shops.

<img width="261" height="74" alt="image" src="https://github.com/user-attachments/assets/40c16e9b-5336-4afa-8bcd-8f2dac4a6660" />
